tools/staticbuild/build_lib.sh

```diff
@@ -34,6 +34,7 @@ $MAKE DEPS_PATH=$DEPS_PATH $PWD/3rdparty/tvm/nnvm/lib/libnnvm.a
 $MAKE DEPS_PATH=$DEPS_PATH PSLITE
 
 if [[ $VARIANT == *mkl ]]; then
+    MKLDNN_LICENSE='license.txt'
```

Comment:
These licenses are included as part of the source LICENSE. There is no need to re-add them